src/components/ComponentPages/NickName/NickNameUI/index.tsx

```diff
@@ -0,0 +1,147 @@
+import { useState } from "react";
+import { Table, Input, Select, Form, Typography, Button, Modal } from "antd";
+import Icon, { PlusCircleOutlined } from "@ant-design/icons";
+import styles from "./NickName.module.scss";
+import messages from "../../../../messages";
+
+const originData = [];
+const { Option } = Select;
+
+for (let i = 0; i < 5; i++) {
+  originData.push({
+    key: i.toString(),
+    srNo: "0" + (i + 1).toString(),
+    nickNameId: `${501 + i}`,
+    nickName: `Amanda ${i}`,
+    visibilityStatus: `Public`,
+  });
+}
+
+export default function NickNameUI({
+  add_edit_form,
+  addEditBtn,
+  addEditTitle,
+  isNickNameModalVisible,
+  editNickName,
+  nickNameForm,
+  handleAddNickName,
+  handleNickNameCancel
+}) {
+
+  const [data, setData] = useState(originData);
```

Comment:
setData is not used

Reply:
This will be used when API will be implemented and we will set the data into the table.